SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -2,17 +2,31 @@ import Foundation
 
 extension String {
 
+    /// Returns a substring based on a subscripted Int range. It is up to the caller to ensure that the
+    /// int range is not out of bounds.
     public subscript(range: Range<Int>) -> String {
         let chars = Array(characters)
         let substringCharacters = chars[range]
         return String(substringCharacters)
     }
+    
+    public subscript(safe range: Range<Int>) -> String {
+        let chars = Array(characters)
+        let substringCharacters = chars[range]
+        return String(substringCharacters)
+    }
```

Comment:
Guessing this is work-in-progress? I'm OK with adding a "safe" access alternative but looks like the implementation here hasn't changed from the original.